pkg/controller/replication.go

```diff
@@ -507,6 +509,66 @@ func (s *Service) DeleteStorageProtectionGroup(ctx context.Context,
 	return &csiext.DeleteStorageProtectionGroupResponse{}, nil
 }
 
+// DeleteLocalVolume deletes a volume on the local storage array upon request from a remote replication controller.
+func (s *Service) DeleteLocalVolume(ctx context.Context,
+	req *csiext.DeleteLocalVolumeRequest) (*csiext.DeleteLocalVolumeResponse, error) {
+	log.Info("Deleting local volume " + req.VolumeHandle + " per request from remote replication controller")
```

Comment:
Are any details/identity of the remote replication controller logged?

Reply:
Not here. We don't have access to information on the remote replication controller from the local replicator sidecar. Replicator can't communicate cross-cluster, only controller can.

The remote replication controller *does* log what it's doing, including setting the annotation that causes this delete, on the remote side.